workfront_bridge/projects/targeted_bonus_media_builder.py

```diff
@@ -145,7 +149,7 @@ def build(self):
         aud.audience_name = '{} audience'.format(self.project_name)
         project.append(aud)
         camp = WFDisplayCampaignBlock()
-        camp.campaign_name = '{} campaign'.format(self.project_name)
+        camp.campaign_name = self.__campaign_name or '{} campaign'.format(self.project_name)
```

Comment:
Why self.__campaign_name instead of self._campaign_name?

Reply:
good catch, that is a mistake